src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+import { Comment } from '../../type/Comment';
+
+import { CommentInfo } from '../CommentInfo';
```

Comment:
```suggestion
import './CommentList.scss';

import { Comment } from '../../type/Comment';
import { CommentInfo } from '../CommentInfo';
```
no need to make empty lines between each import